code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -311,6 +311,30 @@
 	for(var/Z in limb_list)
 		. += regenerate_limb(Z, noheal)
 
+/mob/living/proc/regenerate_organs()
```

Comment:
I had the same thought, honestly. If I had to give any reasoning, regenerate_limbs is in the same file and has a similar purpose to regenerate_organs.